distance_val.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import os
+import pandas as pd
+from ml_collections import config_dict
+import yaml
+
+os.environ["KERAS_BACKEND"] = "jax"
+
+# Note that keras should only be imported after the backend
+# has been configured. The backend cannot be changed once the
+# package is imported.
+import keras
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from wake_vision_loader import get_distance_eval
+
+def distance_val(model_cfg):
```

Comment:
Should we have a similar file for all the other validation and test sets?

Reply:
I think having a file to run each eval on a trained model is useful

Reply:
I'm on it atm :)